src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types';
+import { getUserById } from '../../services';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const user = getUserById(todo.userId);
```

Comment:
If `getUserById` is an asynchronous function, it should be handled using `useEffect` and state to manage the user data. Ensure that `getUserById` is synchronous or handle it asynchronously if needed.